examples/text.md

```diff
@@ -0,0 +1,59 @@
+## Logging Text Values
+
+These examples help you to understand SummaryWriter::Text API in depth
+
+### 0. API 
+
+  1. [Log a Text value](#1-text-value)
+
+### 1. Text Value
+
+A text value could be logged using the following API:
+
+```cpp
+void SummaryWriter<Filewriter>::Text(const std::string &tag,
+                                     int step,
+                                     const std::string& text,
+                                     Filewriter& fw)
+```
+
+The API accepts `tag`, `step`, `value` (String value) and `Filewriter` object.
+
+Following is a snippet that would log some text values.
+
+```cpp
+#include <mlboard/mlboard.hpp>
+#include <iostream>
+#include <chrono> 
+#include <ctime> 
+#include <future>
+using namespace std;
+using namespace mlboard;
+int main()
+{
+    // Creating a FileWriter object that is responsible for logging the summary.
```

Comment:
Same over here I know this is just an example so optional. But In this also four spaces are there.